src/blindBackrunDebug.sol

```diff
@@ -46,7 +46,7 @@ contract BlindBackrun is Ownable {
         address firstPairAddress,
         address secondPairAddress,
         uint percentageToPayToCoinbase
-    ) external onlyOwner {
+    ) public {
```

Comment:
While dropping the `onlyOwner` was necessary for the `blindBackrunFL` to call it, now anyone can call this, and using re-entrancy, could drain WETH balances that accumulate on this. However, there shouldn't be any WETH balances that accumulate, but i feel like that risk is still too high. Perhaps we could change this to `internal` ?

Reply:
Made this change, separating into two contracts, one powered by owner (and holds balance) and one that uses flashloan (and should not hold balance)